### DISCUSSION TITLE
Lesson 6 test_fund_me timestamp 5:49:45

### DISCUSSION THREAD

**Discussion by santoroeth** [smartcontractkit/full-blockchain-solidity-course-py#1242] [Q&A] [answered]
hello, I have this error when running the test "test_fund_me"

can you help me, please

![image](https://user-images.githubusercontent.com/97750911/157672705-dc5e264f-1968-4aa4-b06b-8c37b95e37ec.png)
![image](https://user-images.githubusercontent.com/97750911/157672817-bd55c129-e2d6-4add-8ee5-0a05770144e7.png)

this is my "test_fund_me.py"

![image](https://user-images.githubusercontent.com/97750911/157672963-3e2d0653-1a71-41f4-9cf3-fdd820b18c25.png)

and my "deply.py"

![image](https://user-images.githubusercontent.com/97750911/157673247-76e99a84-3e2a-49f4-b2dd-cd3616e9eac6.png)

thanks.

**Comment by cromewar** [comment, marked as answer]
Hello @santoroeth  in tx2 you have the following;
`tx2 = fund_me.fund_me.withdraw()` this is a typo you must not use `fund_me` twice.
I hope this info might help.

**Reply by santoroeth** [reply]
Hi @cromewar yes, thank you very much, I discovered it last night!


thank you very much for replying again!!
